docs/eventing/sources/apiserversource/reference.md

```diff
@@ -0,0 +1,199 @@
+# ApiServerSource reference
+
+![version](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
+
+This topic provides reference information about the configurable fields for the
+ApiServerSource object.
+
+
+## ApiServerSource
+
+An ApiServerSource definition supports the following fields:
+
+| Field | Description | Required or optional |
+|-------|-------------|----------------------|
+| [`apiVersion`][kubernetes-overview] | Specifies the API version, for example `sources.knative.dev/v1`. | Required |
+| [`kind`][kubernetes-overview] | Identifies this resource object as an ApiServerSource object. | Required |
+| [`metadata`][kubernetes-overview] | Specifies metadata that uniquely identifies the ApiServerSource object. For example, a `name`. | Required |
+| [`spec`][kubernetes-overview] | Specifies the configuration information for this ApiServerSource object. | Required |
+| `spec.mode` | EventMode controls the format of the event. Set to `Reference` to send a `dataref` event type for the resource being watched. Set to `Resource` to send the full resource lifecycle event. Defaults to `Reference`. | Optional |
```

Comment:
```suggestion
| `spec.mode` | EventMode controls the format of the event. Set to `Reference` to send a `dataref` event type for the resource being watched. In this case, only a reference to the resource will be included in the event payload. Set to `Resource` to have the full resource lifecycle event in the payload. Defaults to `Reference`. | Optional |
```

I added some comments about this here: https://github.com/knative/docs/pull/3812/files#r652771165
Sorry, I may not be the best for the wording.